### PR TITLE
#164069213 Integrate Author's Heaven app  coveralls and coverage

### PR DESCRIPTION
**Pull request by BagzieGracious**
#### What does this PR do?
* Integrating coveralls and coverage with Author's Heaven.

#### Description of Task to be completed?
* Added a badge to a README.md file.
* Added a token to a .coveralls.yml file
* Updated requirements.txt file with coverage==4.5.2, coveralls==1.6.0

#### How should this be manually tested?
* click [link](https://coveralls.io/builds/21965188) to view test coverage

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
* [#164069213](https://www.pivotaltracker.com/story/show/164069213)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Coverage Status](https://coveralls.io/repos/github/andela/Ah-backend-aquaman/badge.svg?branch=master)](https://coveralls.io/github/andela/Ah-backend-aquaman?branch=master)
```
Comment (CryceTruly) [inline README.md]: Thanks, @BagzieGracious ,I think you want to add the branch as develop as that's the one we will be merging to.

**Review by Peace-Apple — DISMISSED** [review]
LGTM

**Review by Peace-Apple — COMMENTED** [review]
Please add a line after your code as per the pep8 standards

**Review by Peace-Apple — DISMISSED** [review]
LGTM

**Review by wasibani-roy — DISMISSED** [review]

**Review by joelethan — DISMISSED** [review]

**Review by joelethan — DISMISSED** [review]
LGTM

**Review by CryceTruly — APPROVED** [review]

**Review by CryceTruly — DISMISSED** [review]

**Review by wasibani-roy — DISMISSED** [review]
LGTM

**Comment by joelethan** [comment]
@BagzieGracious In the commit messages, you are not leaving a line between the header, body and footer.

**Review by CryceTruly — DISMISSED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (joelethan) [reply README.md]: Yeah, that is True

**Review by wasibani-roy — CHANGES_REQUESTED** [review]
In your PR under the how you test it section, I think it would be more prudent to maybe have a link to coveralls which when clicked on will lead people to coveralls where they will see a report relating to the repo. I think this is a better way of testing the chore for integrating coveralls

**Review by wasibani-roy — DISMISSED** [review]

**Review by CryceTruly — DISMISSED** [review]
LGTM

**Comment by Peace-Apple** [comment]
Great work, however, the commit message seems to be repeated three times, could you amend the commit and remove the repeated statements.

**Review by Peace-Apple — APPROVED** [review]
LGTM

**Review by CryceTruly — APPROVED** [review]
LGTM

**Review by joelethan — APPROVED** [review]
LGTM

**Pull request merged**